test/e2e/data/infrastructure-vsphere-govmomi/main/clusterclass-runtimesdk/patch-test-extension.yaml

```diff
@@ -3,9 +3,9 @@
   value:
   - name: test-patch
     external:
-      generateExtension: generate-patches.k8s-upgrade-with-runtimesdk
-      validateExtension: validate-topology.k8s-upgrade-with-runtimesdk
-      discoverVariablesExtension: discover-variables.k8s-upgrade-with-runtimesdk
+      generateExtension: generate-patches.${RUNTIME_SDK_EXTENSIONCONFIG_NAME:=k8s-upgrade-with-runtimesdk}
```

Comment:
Wondering if we should use EXTENSIONCONFIG_NAME to be compatible with the env var the core CAPI test is setting